### PR TITLE
Update Vendor Deps and Project to 2021 WPILib Beta

### PR DESCRIPTION
**Pull request by CoolSpy3**

### CODE REVIEW

**Review by alexander-mcdowell — APPROVED** [review]
Looks fine. I don't think merging this will be a problem.

File: build.gradle [hunk 1]
```diff
@@ -39,20 +36,19 @@ deploy {
...
+
 repositories {
     flatDir {
         dirs 'libs'
     }
 }
 
 // Set this to true to enable desktop support.
-def includeDesktopSupport = true
+def includeDesktopSupport = false
```
Comment (brettle) [inline build.gradle]: Seems like the above would disable simulation which is proabably not what we want.

**Review by brettle — CHANGES_REQUESTED** [review]
In addition to the change mentioned, there are also conflicts that need to be resolved.

File: build.gradle (hunk 1 above)
Comment (CoolSpy3) [reply build.gradle]: I didn't notice any problems while testing it, however, I have changed it back to true

**Review by brettle — APPROVED** [review]

**Pull request merged**